components/contribute-cards/Contribute.js

```diff
@@ -50,9 +50,18 @@ const CoverImage = styled.div`
   border-radius: 16px 16px 0 0;
 
   ${props => {
+    const image = props.image ? `url(${props.image}), ` : '';
+
+    if (props.customPrimaryColor) {
+      return css`
+        background: ${image} ${props.customPrimaryColor};
```

Comment:
I would argue that we still want the radial effect to be used in this case, the only difference is that we want to use the project's primary color instead of the parent's one

Reply:
@Betree To use the radial effect we will need to normalize the primary color here first. As we do not want to generate the whole theme again I think we can borrow some code from the `getTheme` method.

https://github.com/opencollective/opencollective-frontend/blob/deff1f8e47bac24859d624a564fe1f1854762892/components/CollectiveThemeProvider.js#L50-L55

Maybe we can move this code to a utility for re-using.

Or, Is there an alternative?

Reply:
Yes, that's what I was suggesting with https://github.com/opencollective/opencollective-frontend/pull/7153#discussion_r754037362. I think that's a good option.